XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/OpenVR/OpenVRDataProvider.cs

```diff
@@ -113,7 +116,7 @@ protected override GenericJoystickController GetOrAddController(string joystickN
         /// <inheritdoc />
         protected override SupportedControllerType GetCurrentControllerType(string joystickName)
         {
-            if (string.IsNullOrEmpty(joystickName) || !joystickName.Contains("OpenVR"))
```

Comment:
This will break WMR controllers

Reply:
I'll retest again but it didn't give me any issues. The Oculus controllers don't have "openVR" in their detected device description.
If there is an issue, I'll revisit the logic to add a special case for Oculus (and any other controller that doesn't expose a device with OpenVR in the description)

Reply:
Ok, this had no effect on Standalone or WMR, so not issue here